### ISSUE TITLE
Capturing the raster data

### ISSUE THREAD

**Issue by news24lor** [3liz/lizmap-web-client#1289]
I'd like some information. The raster in QGIS server can be queried pixel by pixel for each band. The data is always a numerical value. I would like to know if within Lizmap there is the possibility to capture the pixel value with a mouse click and process it to become a URL. Basically an option in Javascript to capture the value of the raster. Is there this possibility in Lizmap?
Thanks again.

**Comment by Gustry** [comment]
@news24lor Did you make some progress on your question?

**Comment by news24lor** [comment]
Hi @Gustry 
I actually bypassed that limitation in this way. My goal was to link a raster cell to some external URLs with graphs and tables with thousands of data to be quickly viewed within webGIS. I did it this way: I created a vector grid as big as the raster not visible on the map where each vector square has several fields with a URL and other data. When I click on the raster the border of the invisible vector square lights up yellow and the window with the various data and URLs opens. I can say that the result is really very fast and was not expected. Of course if in the future also the data of the raster layer could be manipulated like the data hooked to the vector layer it would be a big step.
